sdk/src/main/java/com/bugsnag/android/ndk/NativeBridge.java

```diff
@@ -271,8 +274,16 @@ private boolean is32bit() {
     private void handleAddBreadcrumb(Object arg) {
         if (arg instanceof Breadcrumb) {
             Breadcrumb crumb = (Breadcrumb) arg;
+            Map<String, String> metadata = new HashMap<String, String>();
+            if (crumb.getMetadata() != null) {
```

Comment:
`getMetaData()` is non-null so this shouldn't be necessary

Reply:
Resolved through #585